src/main/java/org/trigon/hardware/phoenix6/talonfx/io/SimulationTalonFXIO.java

```diff
@@ -44,10 +45,8 @@ protected void setPosition(double positionRotations) {
     public void applyConfiguration(TalonFXConfiguration configuration) {
         configuration.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
         configuration.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.RotorSensor;
-        if (configuration.Feedback.RotorToSensorRatio != 1.0) {
-            configuration.Feedback.SensorToMechanismRatio *= configuration.Feedback.RotorToSensorRatio;
-            configuration.Feedback.RotorToSensorRatio = 1.0;
-        }
+        configuration.Feedback.SensorToMechanismRatio *= configuration.Feedback.RotorToSensorRatio;
+        configuration.Feedback.RotorToSensorRatio = 1.0;
```

Comment:
separate into a function with a good jdoc

Reply:
Can you explain it? It's hard to write a jdoc when I don't understand it myself

Reply:
These are just settings that don't work well with sim:
Inverted does some weird things, and shouldn't really be used. Also isn't necessary in any case. 
The feedback sensor is always the rotor to avoid delay/bugs from using a cancoder that derives its positions from the motor itself - this also explains why no rotorToMechanismRatio, it's only a remote encoder setting.
But then, if someone did set a rotorToMechanismRatio, we want to use it as the conversions ratio and the normal gear ratio, so we multiply sensorToMech and then make sure rotorToSensor is 1.